Automattic/wp-calypso
# Cleaning code for Email Stats

#### Proposed Changes

This PR just cleans some functions left behind when passing them to the correct file.

#### Testing Instructions

1. Apply this branch to your local Calypso repository and start the application.
2. Insert manually in the browser address bar an URL with this format: https://calypso.localhost:3000/stats/email/opens/[the-domain-of-your-blog]/day/[post-id]?flags=newsletter/stats. Note that you can change day for hour.
3. Check that it shows the right information in the chart and the other cards.
4. Then insert manually http://calypso.localhost:3000/stats/day/[the-domain-of-your-blog]?flags=newsletter/stats
5. Check that the email card is still showing the right data.
